src/connected/autobinding.ts

```diff
@@ -237,10 +234,11 @@ export class AutoBindingService {
         await this.bindingService.saveBinding(
           bindingSuggestion.sonarProjectKey, bindingSuggestion.connectionId, unboundFolder);
         break;
-      case CHOOSE_MANUALLY_ACTION:
+      case CHOOSE_MANUALLY_ACTION: {
```

Comment:
Why are the curly braces added? I'm not familiar with TypeScript but I would guess that it is better for cases where multiple statements will follow before the break.
This is also the same question for the braces added below.

On the other hand, I could also guess it is to have the variable `targetConnection` in the local scope only to be discarded outside of the case statement.

Reply:
https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&fixedInPullRequest=538